targets/TARGET_Freescale/TARGET_KLXX/TARGET_KL25Z/device/TOOLCHAIN_ARM_MICRO/MKL25Z4.sct

```diff
@@ -1,4 +1,16 @@
 
+#if !defined(MBED_BOOT_STACK_SIZE)
+  #define MBED_BOOT_STACK_SIZE 0x400
+#endif
+
+/* Sizes */
+#if (defined(__stack_size__))
+  #define Stack_Size                   __stack_size__
+#else
+  #define Stack_Size                   MBED_BOOT_STACK_SIZE
+#endif
+
+
```

Comment:
Stack size is not used in this script. Do we need sections for HEAP and STACK?

Reply:
Yes we need heap and stack sections, but PR for ARM_MICRO changes is https://github.com/ARMmbed/mbed-os/pull/9766. Should revert change for this file here